pg_ready function requires psycopg2, but not found in flit.ini as an install requirement

When trying to use the pg_ready function to wait for a postgres container to be ready, I got the following error:
```
>       import psycopg2
E       ModuleNotFoundError: No module named 'psycopg2'
```

It looks like `psycopg2` is a requirement for dev, but not for a regular installation:
```
requires=docker
    typing; python_version < '3.5'
dev-requires=sqlalchemy
             sqlalchemy_utils
             sqllogformatter
             psycopg2
             pytest
             redis
```

--- a/flit.ini
+++ b/flit.ini
@@ -15,9 +15,9 @@ classifiers = License :: OSI Approved :: MIT License
 description-file=README-pypi.rst
 requires=docker
     typing; python_version < '3.5'
+    psycopg2
 dev-requires=sqlalchemy
-             sqlalchemy_utils
-             sqllogformatter
-             psycopg2
-             pytest
-             redis
+    sqlalchemy_utils
+    sqllogformatter
+    pytest
+    redis

